src/pretix/control/templates/pretixcontrol/user/2fa_confirm_totp.html

```diff
@@ -37,7 +37,38 @@
         </li>
         <li>
             {% trans "Add a new account to the app by scanning the following barcode:" %}
-            <div class="qrcode-canvas" data-qrdata="#qrdata"></div>
+          <div class="qrcode-canvas" data-qrdata="#qrdata"></div>
```

Comment:
I'm not quite sure how it happened, but this indentation doesn't seem to be a multiple of four :wink: